applications/CompressiblePotentialFlowApplication/custom_processes/define_embedded_wake_process.cpp

```diff
@@ -84,6 +85,21 @@ void DefineEmbeddedWakeProcess::ComputeDistanceToWake(){
     KRATOS_CATCH("");
 }
 
+void DefineEmbeddedWakeProcess::SetWakeDistancesSignAccordingToNormal(Element& rElement, BoundedVector<double, 3>& rElementalDistances) {
+    const bool is_wake_element = PotentialFlowUtilities::CheckIfElementIsCutByDistance<2,3>(rElementalDistances);
+    auto& r_geometry = rElement.GetGeometry();
```

Comment:
```suggestion
    const auto& r_geometry = rElement.GetGeometry();
```
I think this can be const.